Assets/ZSSRichTextEditor.js

```diff
@@ -274,8 +272,12 @@ ZSSEditor.getSelectedText = function() {
 ZSSEditor.getCaretArguments = function() {
     var caretInfo = this.getYCaretInfo();
     
-    this.caretArguments[0] = 'yOffset=' + caretInfo.y;
-    this.caretArguments[1] = 'height=' + caretInfo.height;
+    if (caretInfo == null) {
+        this.caretArguments = null;
```

Comment:
PS: the reason it's good to keep `this.caretArguments` as an array is that there were some performance issues related to continuously re-creating that array, so I ended up reusing the object.